tests/functional/job-submission/19-platform_select.t

```diff
@@ -17,62 +17,37 @@
 #-------------------------------------------------------------------------------
 # Test recovery of a failed host select command for a group of tasks.
 . "$(dirname "$0")/test_header"
-set_test_number 7
+set_test_number 5
 
 create_test_global_config "
 [platforms]
     [[test platform]]
-        hosts = hostname
-
-    [[improbable platform name]]
         hosts = localhost
-        install target = localhost
 "
 
 install_workflow "${TEST_NAME_BASE}"
 
 run_ok "${TEST_NAME_BASE}-validate" cylc validate "${WORKFLOW_NAME}"
-run_fail "${TEST_NAME_BASE}-run" \
+run_ok "${TEST_NAME_BASE}-run" \
     cylc play --debug --no-detach "${WORKFLOW_NAME}"
 
-declare -A GREP_TESTS
+logfile="${WORKFLOW_RUN_DIR}/log/workflow/log"
 
-# Check that we are warned that platform check will not occur until job-submit
-errname='warn cannot check at validate'
```

Comment:
@MetRonnie  removed this warning.

Reply:
Oliver actually - https://github.com/cylc/cylc-flow/pull/3791#discussion_r483692726